### ISSUE TITLE
Feature Request - Support go install command

### ISSUE THREAD

**Issue by DanSM-5** [TwoBitCoders/jx#6]
**Is your feature request related to a problem? Please describe.**

In some projects it is possible to use the go install command

```bash
go install github.com/<user>/<project>@<revision>
```

However `jx` fails because the go.mod file is setting the module as [`example.com/jx`](https://github.com/TwoBitCoders/jx/blob/c371bbc74b32f281601f13352d1a52340a55ae40/go.mod#L1C1-L1C22) causing this issue

```bash
go install github.com/TwoBitCoders/jx@latest
go: downloading github.com/TwoBitCoders/jx v0.0.7
go: github.com/TwoBitCoders/jx@latest: version constraints conflict:
        github.com/TwoBitCoders/jx@v0.0.7: parsing go.mod:
        module declares its path as: example.com/jx
                but was required as: github.com/TwoBitCoders/jx
```

**Describe the solution you'd like**

I think the [go.mod](https://github.com/TwoBitCoders/jx/blob/main/go.mod) file needs to be updated to this:

```diff
- module example.com/jx
+ module github.com/TwoBitCoders/jx
```
I'm not a go dev, so sorry if this is not as simple as this change.

**Describe alternatives you've considered**

Clone and build from source. Understandable but I think the go install command is a nicer approach if you just want to install the package.

**Comment by mftb0** [comment]
Fixed in d675ce9. Thank you for this excellent report!

**Comment by d-enk** [comment]
pls update [installation](https://github.com/TwoBitCoders/jx?tab=readme-ov-file#installation) instruction
